tests/test_resolver.py

```diff
@@ -179,33 +179,7 @@ async def unknown_addrinfo(*args: Any, **kwargs: Any) -> List[Any]:
 
 
 async def test_close_for_threaded_resolver(loop) -> None:
-    resolver = ThreadedResolver(loop=loop)
```

Comment:
@Dreamsorcerer did you miss that this removes an unrelated test?

Reply:
@alexmac it seems like the conflict resolution hasn't been completed correctly and this removed a legit test. Why?

Reply:
Yeah, I managed to find that. It'd be much more obvious (== less mental overhead) if this was explained explicitly.

As for a separate PR — this is the way. Patchback would also work to port from 3.10 to 3.9. The thing is that the `git cherry-pick -x` is supposed to contain a copy of another change, it even generates a reference `(cherry picked from commit xxxxxx)`. By making arbitrary changes, that makes it hard to track, breaking transparency of the updates to Git. Atomic commits/PRs is a convention that is designed around certain expectations/promises. Breaking them hurts future git paleontology, making the introspection harder. This effect is often not visible/obvious to the commit authors, making it a hard sell, but is quite important for keeping sane history.